components/match2/commons/match_group_display_matchlist.lua

```diff
@@ -26,8 +26,7 @@ MatchlistDisplay.configFromArgs = function(args)
 		attached = Logic.readBoolOrNil(args.attached),
 		collapsed = Logic.readBoolOrNil(args.collapsed),
 		collapsible = not Logic.readBoolOrNil(args.nocollapse),
-		---@diagnostic disable-next-line: param-type-mismatch
-		width = tonumber(string.gsub(args.width or '', 'px', ''), nil),
+		width = tonumber((string.gsub(args.width or '', 'px', ''))),
```

Comment:
Make sure to test this! but with `px` suffix and without

Reply:
that was the plan^^